### PR TITLE
@types/Chrome - chrome.system.display updates

### PR DESCRIPTION
**Pull request by niikoo**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not provide its own types, and you can not add them.
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://developer.chrome.com/apps/system_display
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by niikoo** [comment]
All build failures fixed

**Comment by niikoo** [comment]
Can someone please review this? Thanks!

**Comment by couven92** [comment]
changes for `types/chrome/index.d.ts` look good :+1:

**Review by minestarks — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 1]
```diff
@@ -49,3 +49,6 @@ npm-debug.log
 
 # yarn
 yarn.lock
+
+# Tools
+types/chrome/doc-generator/
```
[the remark this answers is not in the document]
Comment (niikoo) [reply .gitignore]: It's reverted now

File: types/chrome/chrome-app.d.ts [hunk 2]
```diff
@@ -1,16 +1,16 @@
 // Type definitions for Chrome packaged application development
 // Project: http://developer.chrome.com/apps/
-// Definitions by: Adam Lay <https://github.com/AdamLay>, MIZUNE Pine <https://github.com/pine613>, MIZUSHIMA Junki <https://github.com/mzsm>, Ingvar Stepanyan <https://github.com/RReverser>, Adam Pyle <https://github.com/pyle>
+// Definitions by: Adam Lay <https://github.com/AdamLay>, MIZUNE Pine <https://github.com/pine613>, MIZUSHIMA Junki <https://github.com/mzsm>, Ingvar Stepanyan <https://github.com/RReverser>, Adam Pyle <https://github.com/pyle>, Nikolai Ommundsen <https://github.com/niikoo>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-/// <reference path="index.d.ts"/>
+/// <reference path="./index.d.ts"/>
```
[the remark this answers is not in the document]
Comment (niikoo) [reply types/chrome/chrome-app.d.ts]: Done

**Comment by niikoo** [comment]
All requested changes fixed

**Review by minestarks — APPROVED** [review]

**Pull request merged**